index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mi CV</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>  
+    <div class="name">
+        <h1>👥 Samuel Guerrero Arcos</h1>
+    </div>
+
+   <div class="CV-Text">
+    <h1>📓 Hoja de vida</H1>
+    </div> 
+
+    <div class="content">
+        <div class="education">
+            <h3>🎓 Educación</h3>
+            <ol>
+                <li>Guaira</li>
+                <p>• Educación pre-Kinder y Kinder</p>
+                <li>Gimnasio Campestre Santa Sofía</li>
+                <p>• Transición</p>
+                <p>• Primaria</p>
+                <p>• Secundaria</p>
+                <li>Universidad de la Sabana</li>
+                <p>• Carrera profesional Ing. Informática</p>
+            </ol>   
```

Comment:
Puede lograr un efecto de lista anidada creando una lista dentro de una lista

```suggestion
            <ol>
                <li>
                    <p>Guaira</p>
                    <ol>
                       <li><p>Educación pre-Kinder y Kinder</p></li>
                    </ol>
                </li>
            </ol>   
```